# RFS Coordinate System and Initial Conditions.
lat0: 37.917929
lon0: -122.331798
yaw0: 0.0
X0:   91.
Y0:   -65.
Psi0: -0.2
V0:   0.0
